Sources/LocalizedStringKit/include/LocalizedStringKit.h

```diff
@@ -26,32 +34,32 @@ NSString *Localized(NSString *_Nonnull value, NSString *_Nonnull comment);
 ///
 /// @param value: The English string
 /// @param comment: String to give context where the value string is used
-/// @param tableName: String to provide additional classification of the value string. Can be used to segment groups of strings in multiple bundles.
+/// @param bundleName: String to provide additional classification of the value string. Can be used to segment groups of strings in multiple bundles.
 ///
 /// Examples
 /// Localized("Cancel", "Action sheet action title", nil)
 /// Localized("Cancel", "Action sheet action title", "primary")
 /// Localized("Cancel", "Action sheet action title", "primary")
-NSString *LocalizedWithTable(NSString *_Nonnull value, NSString *_Nonnull comment, NSString *_Nonnull tableName);
+NSString *LocalizedWithBundle(NSString *_Nonnull value, NSString *_Nonnull comment, NSString *_Nonnull bundleName);
```

Comment:
I'd prefer "WithBundleName" since you're not passing in an `NSBundle` instance?

```suggestion
NSString *LocalizedWithBundleName(NSString *_Nonnull value, NSString *_Nonnull comment, NSString *_Nonnull bundleName);
```

Reply:
updated